_rules/SC1-2-1-video-only-and-audio-only-alternative.md

```diff
@@ -0,0 +1,221 @@
+---
+name: audio only and video only has an accessibile alternatives
+rule_type: composite
+description: |
+  This rule checks if audio only elements and video elements without audio have an alternative available
+
+success_criterion:
+- 1.2.1 # Audio-only and Video-only (Prerecorded)
+
+atomic_rules:
+- SC1-2-1-audio-transcript
+- SC1-2-1-media-alternative-audio
+- SC1-2-1-media-alternative-video
+- SC1-2-1-video-only-description-track
+- SC1-2-1-video-only-element-transcript
+
+authors:
+- Wilco Fiers
+- Brian Bors
+---
+
+## Test Procedure
+
+### Applicability
+
+The rule applies to any [non-streaming](#non-streaming) `video` element [visible on the page](#visible-on-the-page), where the video doesn't contain audio.
+
+AND
+
+The rule applies to any [non-streaming](#non-streaming) `audio` element [visible on the page](#visible-on-the-page).
```

Comment:
See below, I think this rule should be split into two -- one for each of these applicability conditions.

Reply:
Has been split.